packages/create-flex-plugin/templates/public/index.html

```diff
@@ -0,0 +1,32 @@
+<!-- IMPORTANT -->
```

Comment:
Why don't you run `flex-check-start` instead? This would then copy over the `index.html`

Reply:
yeah good idea